addon/data/shooter-interactive-worker.js

```diff
@@ -873,7 +873,9 @@ function autoSelect(ids) {
   if (pos.top === null && pos.bottom === null &&
       pos.left === null && pos.right === null) {
     console.warn("Auto-detect selection failed: No suitable element found");
-    showHelpMessage("Auto-detect selection failed. Selected visible area.");
+    setTimeout(function() {
+      showHelpMessage("Cannot detect selection automatically.");
+    }, 500);
```

Comment:
In inline-selection.scss, there's a set of rules for `.pageshot-hide-selection`, a class that is added just before screenshotting, and causes any interface elements to be hidden.  So if the help message gets a class and some rules, it won't need this delay.